clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/AuditLoggingServerInterceptor.java

```diff
@@ -207,6 +206,7 @@ public void onHalfClose() {
 
   <ReqT, RespT> void auditLog(
       Selector selector,
+      Struct auditLogReqCtx,
```

Comment:
Instead of adding a new parameter, can we just set the `auditLogReqCtx` in the `logBuilder`?

Reply:
Sure, as metadata?

Reply:
As context: https://github.com/abcxyz/lumberjack/blob/main/protos/v1alpha1/audit_log_request.proto#L89
Essentially what you did in the method: `builder.setContext(auditLogReqCtx);`

Reply:
hum...I don't see the context field in logBuilder (AuditLog.Builder). If to set it in the AuditLogRequest, since this request is initialized in the auditLog method, have to pass it in either as a parameter or in the logBuilder (AuditLog.Builder) as metadata I assume.

Reply:
discussed offline, agreed to keep it as it is.